### PR TITLE
PROD-1410 #414-Theme-issue Fix PHP: Add Filter to add style to link tag

### PR DESCRIPTION
**Pull request by kishanjasani**
### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes https://github.com/buddyboss/buddyboss-theme/issues/414

### How to test the changes in this Pull Request:

1. Go to forum pages 
2. Check the sub forum styles are consistent.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### CODE REVIEW

File: src/bp-forums/forums/template.php [hunk 1]
```diff
@@ -884,6 +884,9 @@ function bbp_list_forums( $args = '' ) {
 				$counts = $r['count_before'] . implode( $r['count_sep'], $count ) . $r['count_after'];
 			}
 
+			// Change the style of link tag.
+			$r['link_before'] = apply_filters( 'bb_forum_link_custom_attr', $r['link_before'], $title );
```
Comment (dipskakadiya) [inline src/bp-forums/forums/template.php:888]: @kishanjasani This filter should be for the entire `$r` not only for link_before. 

```$r = apply_filters( 'bb_forum_link_custom_attr', $r, $title );```

**Review by dipskakadiya — CHANGES_REQUESTED** [review]

File: src/bp-forums/forums/template.php (hunk 1 above)
Comment (kishanjasani) [reply src/bp-forums/forums/template.php]: Thanks for the review. Did changes as per suggestion.

**Review by dipskakadiya — APPROVED** [review]

File: src/bp-forums/forums/template.php [hunk 2]
```diff
@@ -892,6 +892,9 @@ function bbp_list_forums( $args = '' ) {
 				$counts = $r['count_before'] . implode( $r['count_sep'], $count ) . $r['count_after'];
 			}
 
+			// Change the style of link tag.
+			$r = apply_filters( 'bb_forum_link_custom_attr', $r, $title );
+
```
Comment (chetansatasiya) [inline src/bp-forums/forums/template.php:897]: @kishanjasani Can't you use any of these filter `bbp_before_list_forums__parse_args` & `bbp_after_list_forums__parse_args` to make it work?

Comment (chetansatasiya) [reply src/bp-forums/forums/template.php:897]: I don't think we need to add a new filter because we have already a filter to change the args.

**Pull request closed without merging**